cdk/src/open-data-platform/frontend/frontend-stack.ts

```diff
@@ -48,6 +48,20 @@ export class FrontendStack extends Stack {
       defaultRootObject: 'index.html',
       priceClass: cloudfront.PriceClass.PRICE_CLASS_100, // Only deploy to NA and EU access points
       enableLogging: true,
+      errorResponses: [
+        // Vue-router requires all URLs to route to index.html. This should not interfere with the
+        // tileserver's URLs, as long as the tileserver does not return 403 or 404.
+        {
+          httpStatus: 403,
+          responseHttpStatus: 200,
+          responsePagePath: '/index.html',
+        },
+        {
+          httpStatus: 404,
+          responseHttpStatus: 200,
+          responsePagePath: '/index.html',
+        },
+      ],
```

Comment:
What would happen if the tile server threw a 404? Is there a way to override this setting for the /tiles/* path?